### PR TITLE
Added TravisCI file

### PR DESCRIPTION
**Pull request by priyansh19**
#7

### CODE REVIEW

**Comment by priyansh19** [comment]
@pombredanne I am unable to understand why this test is failing. Please help me out

**Comment by priyansh19** [comment]
Added .travis.yml file #7

**Comment by pombredanne** [comment]
Thanks... We need to enable Travis on this repo then too, right?

**Comment by priyansh19** [comment]
Yes

**Comment by priyansh19** [comment]
Till now I have added basic TravisCI file for testing 
I will improve it in future

**Comment by priyansh19** [comment]
Can I apply for contributing this project in GSOC based on my current contributions ?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: python
+python:
+ - "stable"
```
Comment (majurg) [inline .travis.yml]: Where are the tests running? There is nothing here.

**Pull request closed without merging**